ran-simulator/values.yaml

```diff
@@ -28,6 +29,10 @@ towerSpacingHoriz: 0.02
 towerSpacingVert: 0.02
 zoom: 13
 
+towerPort:
+  numTowers: 9
```

Comment:
this is calculated from inside the `ran-simulator` as `towerCols` x `towerRows` - it should not be specified here
https://github.com/onosproject/ran-simulator/blob/c17434a548562cb524bdffa7eeead4e739a7c5eb/cmd/trafficsim/trafficsim.go#L165

Reply:
Yup, that make sense. Let me fix it.

Reply:
Removed as well.